browser/net/brave_torrent_redirect_network_delegate_helper.h

```diff
@@ -20,6 +20,10 @@ class URLRequest;
 
 namespace webtorrent {
 
+bool ShouldRedirectRequest(
```

Comment:
can you add a comment please, the condition inside is impossible to human read

Reply:
Added :smile: